### PR TITLE
zephyr-dom0-xt: enable ci build test

### PR DESCRIPTION
**Pull request by GrygiriiS**
Enable Dom0 build test for targets:
 rpi_5,
 rcar_spider_ca55,
 rcar_salvator_xs_m3,
 rcar_h3ulcb_ca57,
 qemu_cortex_a53

**Note.** It's only a build test. Dom0 execution on all above platforms (except rpi5) is not verified.

### CODE REVIEW

File: prj.conf [hunk 1]
```diff
@@ -8,3 +8,4 @@ CONFIG_SHELL_STACK_SIZE=4096
 
 CONFIG_LOG=y
```
Comment (oleksiimoisieiev) [inline prj.conf:9]: neither build nor work....

**Review by oleksiimoisieiev — APPROVED** [review]
+1 with a small comment
`Acked-by: Oleksii Moisieiev <oleksii_moisieiev@epam.com>`

**Comment by oleksiimoisieiev** [comment]
This PR has my ack. @firscity please take a look

**Comment by firscity** [comment]
> Note. It's only a build test. Dom0 execution on all above platforms (except rpi5) is not verified.

In such case, I don't see much sense in building platforms, that were not initially tested. Or at least it should be mentioned in commit message, docs and files related to mentioned platforms to not make users confused.

**Pull request closed without merging**